ubi_manifest/worker/tasks/content_audit.py

```diff
@@ -194,6 +194,17 @@ def _latest_input_mdds(fts: list[Future[set[UbiUnit]]]) -> list[UbiUnit]:
     return out
 
 
+def _rpm_match(out_rpm: UbiUnit, in_rpm: UbiUnit) -> bool:
+    if (out_rpm.name, out_rpm.arch) != (in_rpm.name, in_rpm.arch):
+        return False
+
+    if f"{out_rpm.release}&{in_rpm.release}".count("module") == 1:
```

Comment:
Although this seems to be a nice approach, but I don't think that you should rely on `module` string in `release`  part as the only indicator that the package is modular or non-modular  I guess this is going to work in some cases but I would say that for modular packages we should probably do different approach.  

But the bigger problem will appear when you have modulemds with the same name but different streams in the repository - you will most likely get false positives.

Example:
* test_module:stream_1 includes foo-rpm.version1
* test_module:stream_2 includes foo-rpm.version2

It'a absolutely correct to have both RPMs present in the repository.

So I would suggest:
- A) skip modular RPM completely from EVR comparison. You get all modular RPMs safely by simple and quick query: https://github.com/release-engineering/ubi-manifest/blob/ef4d440cf1b623105d1d3d1be1e34f3e658352bd/ubi_manifest/worker/tasks/depsolver/rpm_depsolver.py#L84
- B) for modular RPMs we should probably test only their presence in output repo.

what I think is important to do at least point A) for this content audit change, the B) maybe filed as follow up issue if you feel its scope is too big now.

Reply:
...but it gets a bit more complicated since modulemd lives in binary repo but in its artifacts list there are binary, debug and source packages listed :)

Reply:
Okay. Thanks, @rbikar. I've updated to skip modular RPMs.